documentation/sync.md

```diff
@@ -1,22 +1,35 @@
 # Syncing commands
```

Comment:
`step_one` or maybe something generic like `eth_syncing`? `step_one` lends itself in my eyes to having docs named as subsequent steps which might be tricky since the next step would be a split choice between the custom transformers or the generic transformer.

Reply:
Yep, I totally agree to a rename.

I agree that having it called `step_one` would make it feel like the rest would need to be named `step_x`.  If we did make that change, then we could rename `custom-transformers` and `generic-transformer` to `step-two-custom-form` and `step-two-generic-transform`, respectively?

Reply:
I renamed it `data-syncing` for now, though I'm not sure that's quite right either. 🤔 

829a581727e7d87e962a97485cd3e3718ae19e11